core/backend/src/IModelSyncDataStore.ts

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+/** @packageDocumentation
+ * @module SQLiteDb
+ */
+
+import { CloudSqlite } from "./CloudSqlite";
+import { VersionedSqliteDb } from "./SQLiteDb";
+
+
+/** @beta */
+export namespace IModelSyncDataStore {
```

Comment:
This namespace should be called "SchemaSync"

Reply:
Fixed